lib/logidze/versioned_association.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+module Logidze
+  module VersionedAssociation
+    def load_target
+      target = super
+
+      return target if inversed
```

Comment:
I'll try to explain it with the example:

when we call `post.comments`, each comment in this association will have its `post` association initialized as inversed. So I thought, that we don't want to perform our operations in this case. I'll check this more closely during specs writing